__tests__/quiz.test.js

```diff
@@ -36,7 +36,7 @@ describe('readCurrentQuestion', () => {
 describe('verifyCurrentQuestion', () => {
   it('returns Game Over if quiz is finished', () => {
     const quiz = new Quiz(jest.fn());
-    quiz.currentQuestionIndex = 8;
+    quiz.currentQuestionIndex = 5;
     expect(quiz.verifyCurrentQuestion()).toBe('Game Over');
   });
   it('verifies the guess against the current question', () => {
```

Comment:
For this failing test, you only have one question in the quiz.
When you call quiz.verifyCurrentQuestion, you move onto the next question, which means `quiz.isFinished` returns `true`.

On line 35 of quiz.js, you are checking `isFinished` and  `isHighScore` - you should be checking `isHighScore()` as it is a function - `isHighScore` on it's own is truthy.

So because the quiz is finished, and `isHighScore` is truthy, the conditional on line 35 is evaluating to true in ths test.

The quiz in the test is instantiated with a list of questions only, so `this.player` is undefined.

On line 37 of quiz.js, you are referring to this.player.name, but this.player is undefined - so the test is throwing the error `Cannot read property 'name' of undefined`

This is the same for a couple of other tests too.